presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentOptions.java

```diff
@@ -33,35 +30,12 @@
     @Option(names = "--without-presto", description = "Do not start presto-master")
     public boolean withoutPrestoMaster;
 
-    @Option(names = "--bind", description = "Bind ports on localhost")
-    public boolean bindPorts = toBoolean(firstNonNull(System.getenv("PTL_BIND_PORTS"), "true"));
+    @Option(names = "--no-bind", description = "Bind ports on localhost", negatable = true)
+    public boolean bindPorts = true;
```

Comment:
What drive what is default value of option here. The value assigned to `bindPorts` variable?
If so can you just declare option ss `@Option(names = "--bind", description = "Bind ports on localhost", negatable = true)`

Reply:
No I can't. It's just the way negatable works (so by default it's --bind, --no-bind will switch to false)